linera-execution/src/wasm/wasmer.rs

```diff
@@ -41,6 +38,8 @@ static CONTRACT_CACHE: Lazy<Mutex<ModuleCache<CachedContractModule>>> = Lazy::ne
 /// A cache of compiled service modules.
 static SERVICE_CACHE: Lazy<Mutex<ModuleCache<Module>>> = Lazy::new(Mutex::default);
 
+static GAS_GLOBAL_NAME: &str = "LINERA_GAS";
```

Comment:
Nit: adding documentation

```suggestion
/// The name of the global variable used to track fuel consumption.
static GAS_GLOBAL_NAME: &str = "LINERA_GAS";
```

Reply:
Thank you!